.evergreen/run-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       AUTH                    Set to enable authentication. Defaults to "noauth"
+#       SSL                     Set to enable SSL. Defaults to "nossl"
+#       MONGODB_URI             Set the suggested connection MONGODB_URI (including credentials and topology info)
+#       MARCH                   Machine Architecture. Defaults to lowercase uname -m
+
+
+AUTH=${AUTH:-noauth}
+SSL=${SSL:-nossl}
+MONGODB_URI=${MONGODB_URI:-}
+TESTS=${TESTS:-}
+PHP_VERSION=${PHP_VERSION:-7.2.21}
+
+# The PLATFORM environment variable is used by the matrix testing project.
+# For the time being, we can use that to install the correct extension version
```

Comment:
I assume `PLATFORM` is actually used for some other purpose. Is there a ticket to introduce another environment variable (e.g. `MATRIX_TESTING`) specifically so scripts can determine if they're running for matrix testing?

Reply:
Yes, this is an ugly monkey patch. `PLATFORM` contains the platform being tested on and is only set by the drivers-matrix-testing pipeline. A more elaborate solution removes the need to call this by being able to define environment variables for driver entries and invoking install-dependencies.sh automatically.

Reply:
After some more playing around, I was able to remove the check on PLATFORM and the hardcoded PHP_VERSION. With recent changes in drivers-matrix-testing, custom variables are passed through, so I can leverage the new `PHP_VERSION` and `PHP_IS_MATRIX_TESTING` variables. Since they can only be set per driver (instead of per driverEra, e.g. "4.4"), I can't use the same method for defining the extension version to install, so that's still hardcoded in run-tests.sh.

Reply:
Should this PR wait on https://github.com/mongodb/drivers-matrix-testing/pull/7 to rename the environment variable to `IS_MATRIX_TESTING`?

Reply:
This has been merged and renamed.